capsicum/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "capsicum"
+edition = "2021"
+version = "0.1.3"
+authors = ["Daniel Robertson <dan.robertson@anidata.org>"]
+license = "MPL-2.0"
+repository = "https://github.com/dlrobertson/capsicum-rs"
+rust-version = "1.62"
+description = """
+Simple intuitive Rust bindings for the FreeBSD capsicum framework
+"""
+build = "build.rs"
+readme = "README.md"
+keywords = ["sandbox", "FreeBSD", "capsicum"]
+categories = ["os::freebsd-apis"]
+
+include = ["src/**/*", "test/**/*", "build.rs", "LICENSE", "README.md", "CHANGELOG.md"]
+
+[package.metadata.docs.rs]
+rustdoc-args = ["--cfg", "docsrs"]
+targets = [
+  "x86_64-unknown-freebsd",
+]
+
+[features]
+casper = [ "casper-sys", "libnv", "libnv-sys" ]
```

Comment:
Thanks for putting this behind a feature. Eventually I think it would make sense to enable this by default.

Reply:
I thought about that.  But bringing in casper-sys means linking to another shared library, which creates hassle for anybody who is cross-compiling.  That's why I left it as non-default.